Userland/Utilities/image.cpp

```diff
@@ -180,7 +180,7 @@ static ErrorOr<void> save_image(LoadedImage& image, StringView out_path, bool pp
         return {};
     }
     if (out_path.ends_with(".png"sv, CaseSensitivity::CaseInsensitive)) {
```

Comment:
This here uses CaseInsensitive; below should probably match?

Reply:
Also, this here uses ".png" with a dot; below should probably match that too.

Reply:
I thought that it would also allow `apng` but these are not even created by `image`, was an half-baked thought.